news/986_input_validation.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* ``openfe quickrun`` now fails up-front when the user-supplied output file (``-o``) is invalid.
```

Comment:
Could you maybe add a bit more detail here on what "invalid" means (i.e. already exists and/or the directory to the file doesn't exist)